examples/legacy/docker-rke2.yaml

```diff
@@ -0,0 +1,245 @@
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: Cluster 
```

Comment:
I think ClusterClass resource should be applied first, otherwise we will get WARNING that Cluster is trying to use non-existing ClusterClass